source/OutfitterPanel.cpp

```diff
@@ -210,17 +209,49 @@ int OutfitterPanel::DetailWidth() const
 
 int OutfitterPanel::DrawDetails(const Point &center)
 {
-	if(!selectedOutfit)
-		return 0;
-	
-	outfitInfo.Update(*selectedOutfit, player, CanSell());
-	Point offset(outfitInfo.PanelWidth(), 0.);
-	
-	outfitInfo.DrawDescription(center - offset * 1.5 - Point(0., 10.));
-	outfitInfo.DrawRequirements(center - offset * .5 - Point(0., 10.));
-	outfitInfo.DrawAttributes(center + offset * .5 - Point(0., 10.));
-	
-	return outfitInfo.MaximumHeight();
+
+	string selectedItem = "Nothing Selected";
+	const Font &font = FontSet::Get(14);
+	const Color &bright = *GameData::Colors().Get("bright");
+
+	int heightOffset = 20;
+
+	if(selectedOutfit)
+	{
+		outfitInfo.Update(*selectedOutfit, player, CanSell());
+		selectedItem = selectedOutfit->Name();
+
+		const Sprite *thumbnail = selectedOutfit->Thumbnail();
+		if(!thumbnail)
+		{
+			thumbnail = SpriteSet::Get("ui/outfitter selected");
```

Comment:
Should we always render the outfit selected image behind the outfit image instead of only when there is no image to show? And same deal with the ships; should we put the ship selected image behind the image of the ship? If you look at the right-most bar, you'll notice that the currently selected escort actually has the selected ship image behind it.

Reply:
I honestly don't care how we do it. This way is fine, the other way is fine, but in all seriousness someone needs to make the call here. Without an actual PO (or PM) at the reins I think it boils down to the testers or at least the most senior developers to make the call, but in any case it is a super easy change to make. Just make the call, what do you think makes the most sense, because I have been play testing this and I definitely feel that it doesn't make a difference and a little input goes a long way here.